src/screens/OrganizationPeople/AddMember.tsx

```diff
@@ -347,9 +348,7 @@ function AddMember(): JSX.Element {
                   <TableHead>
                     <TableRow>
                       <StyledTableCell>#</StyledTableCell>
-                      <StyledTableCell align="center">
-                        User Name
-                      </StyledTableCell>
+                      <StyledTableCell align="center">User</StyledTableCell>
```

Comment:
@krishna619 the `User` translation is still missing

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/52276473/fed43385-977d-41ef-b352-636a6b77bf1c)
@beingnoble03 
added translation for user and add member , please review.